docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -0,0 +1,173 @@
+which the set fields `roles` and `name`.
+
+### Session kinds
+
+Require and allow policies have to specify which sessions they apply to. Valid options are `ssh` and `k8s`.
+
+- `ssh` policies apply to all SSH sessions towards a node running the Teleport SSH server.
+- `k8s` policies apply to all kubernetes sessions to clusters connected to teleport.
+
+### Participant modes
+A participant joining a session will always have one of three modes:
+
+- `peer`: Can join and collaborate in a session. They can view output and send input.
+- `moderator`: Can join and watch a session. They can view output and forcefully terminate the session at will.
+- `observer`: Can join and watch a session. They cannot control the session in any way.
+
+When joining a session with `tsh join` or `tsh kube join`, a user can specify a mode with the `--mode <mode>` flag
+, where the mode is one of `peer`, `moderator` or `observer`. By default, the mode is `peer` for SSH and
+`moderator` for Kubernetes sessions.
+
+A participant may leave a session with the shortcut `c` while in observer or moderator mode.
+When in moderator mode, a participant may also forcefully terminate the session at any point in time
+with the shortcut `t`.
+
+### Require policy count
+
+Require policies can have a variable amount of users that need to match the filter expression
+in order to satisfy the policy. The `count` field of a require policy is a positive integer
+value that specifies the minimum amount of users this policy requires.
+
+## Backwards compatibility with Server Access
+
+Previously, Server Access did not include controls over which users can join a session.
+To work around this, RBAC rules are ignored for users that only have V4 roles (`version: v4` in the role specification).
+New roles are created as V5. V4 roles are upgraded when they are modified in the UI.
+If a user has any attached V5 roles (`version: v5` in the role specification), the new RBAC access checks will be enforced.
+
+## MFA-based presence
+
+When `per_session_mfa` is set to `true` via [role or cluster settings](../../access-controls/guides/per-session-mfa.mdx), Teleport enforces
+MFA-based presence checks for moderators.
+This requires that all moderators wishing to join have a configured U2F or WebAuthn MFA token.
+
+Every 30 seconds, Teleport will issue a prompt to the user in the terminal, asking them
+to press their MFA token in the next 15 seconds. This will happen continously during the session
+and exists so that moderators are always present and watching a given session.
+
+If no MFA input is received within 60 seconds, the user is kicked
+from the session which may pause it, if RBAC policies are no longer met.
+
+## Session invites
+
+Moderatated Sessions tracks sessions via `SessionTracker` resource. When starting
+an interactive SSH or Kubernetes session using `tsh ssh` or `tsh kube exec` respectively,
```

Comment:
I wasn't sure how the mention of the `SessionTracker` resource relates to the CLI guidance in the next sentence.